framework/ui/src/components/organisms/vc-table/_internal/vc-table-cell/vc-table-cell.vue

```diff
@@ -17,6 +17,11 @@
     {{ value.toLocaleDateString() }}
   </div>
 
+  <!-- Finished date cell -->
```

Comment:
'Finished' can't be considered as primitive types such as other described here date-ago and money. So better to define a custom template or better extend the existing cell type 'date-ago` with a `format` parameter.